client/components/checklist/task.js

```diff
@@ -196,6 +199,11 @@ class Task extends PureComponent {
 									>
 										{ taskActionButtonText }
 									</Button>
+									{ ! completed && showSkip ? (
```

Comment:
The ternary isn't necessary in this case, you could avoid it if you wanted to by using a `&&`, e.g.

`{ ! completed && showSkip && <Button {...} /> }`

Reply:
aah yes. I keep forgetting that you can do this. Thanks for the suggestion. It's been updated.